mteb/tasks/Classification/ara/RestaurantReviewSentimentClassification.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class RestaurantReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="RestaurantReviewSentimentClassification",
+        dataset={
+            "path": "ar_res_reviews",
+            "revision": "d51bf2435d030e0041344f576c5e8d7154828977",
+        },
+        description="Dataset of 8364 restaurant reviews from qaym.com in Arabic for sentiment analysis",
+        reference="https://link.springer.com/chapter/10.1007/978-3-319-18117-2_2",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
```

Comment:
I see in the hub that there is only a `train` split for this dataset, loading a `test` split will raise an error imo... How did you manage to get the benchmark run on the `test` split?

Reply:
Hi @imenelydiaker, I did notice that and I used the `train` split at first. But since the data loading function applies the data transform after loading the dataset from the hub, the evaluation uses the `train` split for testing (which is what I've encountered). Here we can specify a `test` split as I've accounted for that and created a train/test split in the dataset transform.

Reply:
This should then be "train" then (to indicate that it is data from the train split on HF that is being used). I believe you use the test train split to downsample the data? 

I would instead shuffle with a seed and select the first 2048 samples.